package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@babel/core": "7.9.0",
     "@svgr/webpack": "4.3.3",
+    "@tehshrike/duration-iso-8601": "^2.0.2",
```

Comment:
This one looks like very small package, I've went through the code and looks pretty minimal.